setup.py

```diff
@@ -6,26 +6,29 @@
 # Author: gdestuynder@mozilla.com
 
 import os
-from distutils.core import setup
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 setup(
     name = "libnfldap",
-	py_modules=['libnfldap'],
-    version = "1.0.5",
+    py_modules=['libnfldap'],
+    version = "1.0.6",
     author = "Julien Vehent",
     author_email = "jvehent@mozilla.com",
     description = ("A client library to generate ipset and iptables rules from LDAP records."),
     license = "MPL",
     keywords = "libnfldap client library",
     url = "https://github.com/mozilla/libnfldap",
     long_description=read('README.rst'),
-	requires=['ldap', 're'],
```

Comment:
I suspect dropping `re` forces use to use python2.7/3+ or something like that. Though I'm fine with that.